app/src/main/java/com/palone/planahead/screens/home/ui/components/FloatingActionButtonAddTask.kt

```diff
@@ -6,11 +6,13 @@ import androidx.compose.material3.ExtendedFloatingActionButton
 import androidx.compose.material3.Icon
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 
 @Composable
-fun FloatingActionButtonAddTask(onClick: () -> Unit) {
+fun FloatingActionButtonAddTask(modifier: Modifier = Modifier, onClick: () -> Unit) {
```

Comment:
It's a convenction in jetpack compose. Every reusable composable function should have this in case of slight change of component apperance without interference with main component (it's used later in EditTaskScreen > Scaffold > FAB)